internal/network/manifests_generator.go

```diff
@@ -295,10 +295,10 @@ data:
       telemeterServerURL: {{.TELEMETER_SERVER_URL}}
 `
 
-	prodServiceBaseURL  = "https://api.openshift.com"
-	stageServiceBaseURL = "https://api.stage.openshift.com"
-	stageTelemeterURL   = "https://infogw.api.stage.openshift.com"
-	dummyURL            = "https://dummy.com"
+	stageServiceBaseURL       = "https://api.stage.openshift.com"
+	integrationServiceBaseURL = "https://api.integration.openshift.com"
+	stageTelemeterURL         = "https://infogw.api.stage.openshift.com"
+	dummyURL                  = "https://dummy.com"
```

Comment:
Not blocking, but how about using stuff from `.invalid` TLD instead of domain that is registered to _someone somewhere_? It should not matter much until we decide to send there (accidentally or not) some data that we should not

Reply:
I don't see any reason why not to use `https://dummy.invalid`.
Anyway, I want to keep this PR minimal as possible - only the bug fix.
What you see is only the indention change but this wasn't inserted in this PR.
@mkowalski 

Thanks for your review BTW, I have learned something new :)

Reply:
@mkowalski @ybettan can one of you open an issue to track this? Or a PR to fix it?
I think this is a great point.

Reply:
I have opened https://github.com/openshift/assisted-service/pull/1993 for that

Reply:
:+1: thanks